### ISSUE TITLE
SIDs conflict with STARs of the same name when typed into the arrivals section

### ISSUE THREAD

**Issue by eliuuk** [zlsa/atc#564]
During the development of my next airport, I've found there's an issue with SIDs and STARs of the same name conflicting. For example, in OMAA, there is an ATUDO departure, with suffixes as required, and an ATUDO arrival, also with suffixes as required.

When you place the STAR into the arrivals section for aircraft, and type in for example, ATUDO.ATUDO.OMAA, then run the game, the game will state on the user output '(callsign) unable to fly SID as we have not been assigned a runway!' and the game is trying to use the departure instead of the arrival.

In simple terms, the game is getting mixed up when there is a SID and a STAR of the same name. When arrivals spawn, and the STAR name is the same as a SID name, the game will attempt to use the SID instead of the STAR.

**Comment by erikquinn** [comment]
The ATC repository is being migrated to it's new home at [https://github.com/openscope/openscope](https://github.com/openscope/openscope), 
and thus, all issues are being closed. If this is still an issue with the latest version of the sim 
(accessible at [http://www.openscope.co](http://www.openscope.co)), or is a feature you still think we are lacking, 
please reopen the issue at the new repo.

*Please note that the vast majority of these issues have been copied to the new repository, or else are covered by other issues created there. See the below screenshot for what it looks like when your issue is known in the new repo:*

![image](https://cloud.githubusercontent.com/assets/5103735/22630816/5a32d794-ebce-11e6-9c46-d61a262e2b23.png)


 Thank you!


Closing this issue.
